sources/HUBComponentWrapper.m

```diff
@@ -282,8 +281,9 @@ - (void)component:(id<HUBComponentWithChildren>)component didStopDisplayingChild
         return;
     }
     
+    HUBComponentWrapper * const childComponent = self.childrenByIndex[@(childIndex)];
```

Comment:
Yeah we need to add some tests to cover that! 😁 Yeah, it's definetly alright to limit the functionality for `UIView`-based components, as long as they continue working as expected 😀